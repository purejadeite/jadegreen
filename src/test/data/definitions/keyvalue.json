{
	"sheets": [
		{
			"comment": "キーバリュー形式",
			"output": true,
			"target": {
			    "name": "keyvalue"
			},
		    "cells": [
		        {"id": "values", "row": 2, "options": [{"type": "keyvalue", "keyId": "key", "valueId": "value"}],
		            "columns": [
		                {"id": "key", "column": 2},
		                {"id": "value", "column": 3}
		            ]
		        }
		    ]
		}
	]
}