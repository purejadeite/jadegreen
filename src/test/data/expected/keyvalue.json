[
    {
        "values": {
            "fileExistFlag": "ファイル有無フラグ",
            "fileName": null,
            "fileNameJp": "ファイル名日本語",
            "filePath": "ファイルパス",
            "lines": "行情報",
            "number1": 1,
            "number2": 10.1,
            "number3": -22.333,
            "number4": 99999999999999999999,
            "number5": 0,
            "boolean1": true,
            "boolean2": false,
            "date": "2016/2/29"
        }
    }
]